am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/base/IntegrationBaseTest.java

```diff
@@ -14,20 +14,35 @@
 @SuppressWarnings("PMD")
 public abstract class IntegrationBaseTest {
 
-    protected static final PostgreSQLContainer db = new PostgreSQLContainer().withUsername("sa").withPassword("");
-    private static Jdbi jdbi;
+    protected static final PostgreSQLContainer db;
+    private static final Jdbi jdbi;
 
-    @BeforeAll
-    static void initDatabase() {
+    static {
+        db = createDatabaseContainer();
         db.start();
-        jdbi = Jdbi.create(db.getJdbcUrl(), db.getUsername(), db.getPassword());
-
         initSchema();
+
+        jdbi = Jdbi.create(db.getJdbcUrl(), db.getUsername(), db.getPassword());
     }
 
     @AfterAll
-    static void destroyDatabase() {
-        db.stop();
+    static void cleanupDatabase() {
```

Comment:
AfterAll triggers after all tests in a class correct? Do we want entries overflowing into tests? Could get messy

Reply:
Was this working as we thought before?

Reply:
Yes it will be executed after each test class. It is not perfect but it works as expected for now due to the fact tests run one by one. I tested it both in IDE and from Gradle.